### PR TITLE
applications: nrf_desktop: Support constant latency

### PR DESCRIPTION
**Pull request by pdunaj**
Allow application to use constant latency interrupts.
This setting may be needed for some configurations.

Jira:DESK-738

Signed-off-by: Pawel Dunaj <pawel.dunaj@nordicsemi.no>

### CODE REVIEW

File: applications/nrf_desktop/src/modules/power_manager.c [hunk 1]
```diff
@@ -289,6 +288,11 @@ static bool event_handler(const struct event_header *eh)
 			k_delayed_work_init(&power_down_trigger, power_down);
 			k_delayed_work_submit(&power_down_trigger,
 					      POWER_DOWN_CHECK_MS);
+
+			if (IS_ENABLED(CONFIG_DESKTOP_POWER_MANAGER_CONSTLAT)) {
+				nrf_power_task_trigger(NRF_POWER_TASK_CONSTLAT);
+				LOG_WRN("Constant latency enabled");
```
Comment (MarekPieta) [inline applications/nrf_desktop/src/modules/power_manager.c]: `LOG_INF` ?

**Comment by pdunaj** [comment]
```

desktop52_base_test Line:223  Assert raised, [00000003] <err> os: ***** MPU FAULT *****

desktop52_base_test Line:223  Assert raised, [00000004] <err> os:   Stacking error (context area might be not valid)

desktop52_base_test Line:223  Assert raised, [00000004] <err> os:   Data Access Violation

desktop52_base_test Line:223  Assert raised, [00000004] <err> os:   MMFAR Address: 0x200082dc

desktop52_base_test Line:223  Assert raised, [00000004] <err> os: r0/a1:  0x00000000  r1/a2:  0x0002be93  r2/a3:  0x00000000

desktop52_base_test Line:223  Assert raised, [00000004] <err> os: r3/a4:  0x0003869f r12/ip:  0x20008308 r14/lr:  0x75646f6d

desktop52_base_test Line:223  Assert raised, [00000005] <err> os:  xpsr:  0x00000001

desktop52_base_test Line:223  Assert raised, [00000005] <err> os: Faulting instruction address (r15/pc): 0x623a656c

desktop52_base_test Line:223  Assert raised, [00000005] <err> os: >>> ZEPHYR FATAL ERROR 2: Stack overflow

desktop52_base_test Line:223  Assert raised, [00000005] <err> os: >>> ZEPHYR FATAL ERROR 2: Stack overflow

desktop52_base_test Line:223  Assert raised, [00000005] <err> os: Current thread: 0x20005784 (unknown)

desktop52_base_test Line:223  Assert raised, [00321928] <err> os: Halting system
```

this is probably caused by this CC310 driver... it was disabled but only for mouse.... @#$%^&*()

**Comment by pdunaj** [comment]
I hope https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1369 will fix CI issues.

**Comment by pdunaj** [comment]
Hi @jhn-nordic , @wiminordic , could you give a green tick?

**Pull request merged**